luarules/gadgets/unit_workertime_boost.lua

```diff
@@ -0,0 +1,67 @@
+function gadget:GetInfo()
+	return {
+		name = "Workertime Multiplier Boost",
+		desc = "Allows units with added UnitDefs to build gadget-defined types of units faster.",
+		author = "SethDGamre",
+		date = "April 2024",
+		license = "Public domain",
+		layer = 1,
+		enabled = true
+	}
+end
+
+-- synced only
+if not gadgetHandler:IsSyncedCode() then return false end
+
+-- is unba com on
+if true and Spring.GetModOptions().unbacom ~= "enabled" then return false end
+Spring.Echo("the script ran?!")
+Spring.Echo(Spring.GetModOptions().unbacom)
```

Comment:
✂️

Reply:
Sloppy of me. Sorry. i'll clean up the script of debug stuff